cfme/configure/access_control.py

```diff
@@ -1256,20 +1266,73 @@ def is_displayed(self):
         )
 
 
-class EditTenantView(TenantForm):
+class EditTenantView(View):
     """ Edit Tenant View """
+    form = View.nested(TenantForm)
     save_button = Button('Save')
     reset_button = Button('Reset')
 
     @property
     def is_displayed(self):
         return (
-            self.accordions.accesscontrol.is_opened and
-            self.title.text == 'Editing Tenant "{}"'.format(self.context['object'].name)
+            self.form.accordions.accesscontrol.is_opened and
+            any([self.form.title.text == 'Editing Tenant "{}"'.format(self.context['object'].name),
+                self.form.title.text == 'Editing Project "{}"'.format(self.context['object'].name)])
         )
+        # used any() because `project` inherits `tenant` and edit page can have any one in name
+
+
+class TenantCollection(BaseCollection):
+    """Collection class for Tenant"""
+
+    def __init__(self, appliance):
+        self.appliance = appliance
+
+    def instantiate(self, name, description=None, parent=None, default=False):
+        return Tenant(collection=self, name=name, description=description, parent_tenant=parent,
+                      _default=default)
+
+    def get_root_tenant(self):
+        return self.instantiate('My Company', default=True)
+
+    def create(self, name, description, parent):
+
+        if isinstance(parent, Project):
+            raise ValueError("Projects cannot have children.")
+
+        tenant = self.instantiate(name, description, parent)
```

Comment:
**REQUIRED**: `create` should always end with `return self.instantiate(...)`.

Reply:
`Add` navigation on `Tenant` didn't seem a good design, so here I first initialized the tenant to navigate to its `Details` and then filled the details for create